spec/data_migrations/update_subject_for_messages_spec.rb

```diff
@@ -16,17 +18,17 @@
     let!(:organization)         { FactoryBot.create(:benefit_sponsors_organizations_general_organization, :with_aca_shop_cca_employer_profile, site: site) }
     let!(:employer_profile)     { organization.employer_profile }
     let!(:benefit_sponsorship)  { employer_profile.add_benefit_sponsorship }
-    let (:message) {FactoryBot.build(:message, subject: 'Welcome to MA Health Link')}
+    let(:message) {FactoryBot.build(:message, subject: 'Welcome to MA Health Link')}
```

Comment:
@eidarus, make `me_demo` have the same code as `disable_shop_base` (normalize it)